exporters/otlp-http/logs/build.gradle.kts

```diff
@@ -10,8 +10,7 @@ otelJava.moduleName.set("io.opentelemetry.exporter.otlp.http.logs")
 
 dependencies {
   api(project(":sdk:logs"))
-
-  implementation(project(":exporters:otlp:common"))
+  api(project(":exporters:otlp:common"))
```

Comment:
I think sticking with implementation is better, even if it means duplicating into some build files

Reply:
We talked about this back [here](https://github.com/open-telemetry/opentelemetry-java/pull/3791#discussion_r737575914), but I've tested it and confirmed that we can get away with implementation. 

Best to keep it as implementation until we actually promote something in common out of internal. Will update the grpc modules as well.